spark-iotdb-connector/src/test/scala/org/apache/iotdb/spark/db/IoTDBTest.scala

```diff
@@ -20,7 +20,6 @@
 package org.apache.iotdb.spark.db
 
 import java.io.ByteArrayOutputStream
-
```

Comment:
We should add some insert test and check whether the result is correct.

Reply:
Yep, I have updated some ITs of insert test.